1 entries in records
Started running calling sequence 1
Traceback (most recent call last):
  File "C:\Users\Swathi\Downloads\Etl_New\ETL_Framework\common_utils\audit.py", line 145, in wrapper
    args and len(args[0]) or 0,
             ^^^^^^^^^^^^
TypeError: object of type 'CallSP' has no len()

Finished d_customer in 0.05 seconds
Finished running calling sequence 1 in 0.055395200004568323 seconds
